subprojects/packagefiles/cfitsio/meson.build

```diff
@@ -0,0 +1,270 @@
+project(
+  'cfitsio',
+  'c',
+  version: '4.5.0',
+  meson_version: '>=1.1.0',
+  license: 'CFITSIO',
+  license_files: ['licenses/License.txt'],
+)
+has_fortran = add_languages(
+  'fortran',
+  native: false,
+  required: get_option('fortran'),
+)
+
+SOVERSION = 10
+
+cc = meson.get_compiler('c')
+libm = cc.find_library('m')
+
+if target_machine.system() != 'windows'
```

Comment:
target_machine is used when the thing you are creating is a cross compiler that will, when built and installed and run, emit platform specific outputs for a platform that isn't the one it is installed for.

Probably you mean host_machine instead, as that is the machine that libcfitsio is being compiled to run on.

Reply:
fixed